CUDADataFormats/StdDictionaries/src/classes_def.xml

```diff
@@ -0,0 +1,14 @@
+<lcgdict>
+    <class name="std::vector<std::byte, cms::cuda::HostAllocator<std::byte, 0>>" />
```

Comment:
Could you remind me, the plan ẁas to eventually get rid of these, right?

(well, eventually likely `HostProduct` and `HeterogneeousSoA` will evolve)